clients/guzzle6-adapter.rst

```diff
@@ -1,2 +1,52 @@
-Guzzle6 Adapter
-===============
+Guzzle 6 Adapter
+================
+
+An HTTPlug adapter for the `Guzzle 6 HTTP client`_.
+
+To install the Guzzle adapter, which will also install Guzzle itself (if it was
+not yet included in your project), run:
+
+.. code-block:: bash
+
+    $ composer require php-http/guzzle6-adapter
+
+Then begin by creating a Guzzle client, passing any configuration parameters you
+like::
+
+    $config = [
+        // Config params
+    ];
+    $guzzle = new GuzzleHttp\Client($config);
```

Comment:
it would be consistent to have a `use` here as well.

we could use as GuzzleClient and then below use as GuzzleAdapter. i think that would make it clearer.